src/xcode/ENA/ENA/Source/Scenes/QRScanner/Tooltip/QRScannerTooltipViewController.xib

```diff
@@ -0,0 +1,117 @@
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="19144"/>
+        <capability name="Named colors" minToolsVersion="9.0"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="QRScannerTooltipViewController" customModule="ENA" customModuleProvider="target">
+            <connections>
+                <outlet property="closeButton" destination="Ai7-Cz-8g7" id="YPy-su-xyM"/>
+                <outlet property="descriptionLabel" destination="Lvf-6A-3Ad" id="Jwj-ZC-CIy"/>
+                <outlet property="titleLabel" destination="8LV-op-Cel" id="3Av-n6-Jwx"/>
+                <outlet property="view" destination="i5M-Pr-FkT" id="sfx-zR-JGt"/>
+            </connections>
+        </placeholder>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view clearsContextBeforeDrawing="NO" contentMode="scaleToFill" id="i5M-Pr-FkT">
+            <rect key="frame" x="0.0" y="0.0" width="219" height="102"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="999" text="Title" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontForContentSizeCategory="YES" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="8LV-op-Cel" customClass="ENALabel" customModule="ENA" customModuleProvider="target">
+                    <rect key="frame" x="16" y="12" width="138" height="18"/>
+                    <constraints>
+                        <constraint firstAttribute="width" relation="lessThanOrEqual" constant="200" id="asg-CZ-u6Y"/>
+                    </constraints>
+                    <fontDescription key="fontDescription" style="UICTFontTextStyleHeadline"/>
+                    <color key="textColor" name="ENA Text Primary 1 Color"/>
+                    <nil key="highlightedColor"/>
+                    <userDefinedRuntimeAttributes>
+                        <userDefinedRuntimeAttribute type="string" keyPath="ibEnaStyle" value="headline"/>
+                    </userDefinedRuntimeAttributes>
+                </label>
+                <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="CUs-sA-78T">
```

Comment:
Looks like the scroll view is partly overlapping the bounds of the button. I don't know, maybe its still enough space to tap the button 🤷‍♂️

<img width="326" alt="Bildschirmfoto 2021-10-01 um 10 12 30" src="https://user-images.githubusercontent.com/1941000/135587571-6feeb2c5-e927-41ea-b749-77b263d446bc.png">

Reply:
I put the button on top so it always stays tappable

Reply:
Good catch Artur :D